src/main/scala/beam/sim/BeamHelper.scala

```diff
@@ -337,10 +338,16 @@ trait BeamHelper extends LazyLogging {
       beamConfig.beam.agentsim.simulationName,
       beamConfig.beam.outputs.addTimestampToOutputDirectory
     )
+
     LoggingUtil.createFileLogger(outputDirectory)
     matsimConfig.controler.setOutputDirectory(outputDirectory)
     matsimConfig.controler().setWritePlansInterval(beamConfig.beam.outputs.writePlansInterval)
 
+    logger.info(s"Starting beam on branch $getBranch at commit $getCommitHash.")
```

Comment:
While this is only done once, to remain consistent with the newer, more performant standard could you use `{}`:

`logger.info("Starting beam on branch {} at commit {}.", getBranch, getCommitHash)`